eventkit_cloud/ui/static/ui/app/components/MapTools/SearchAOIToolbar.js

```diff
@@ -100,22 +110,35 @@ export class SearchAOIToolbar extends Component {
                         options={this.state.suggestions}
                         onChange={this.handleEnter}
                         placeholder={'Search admin boundary or location...'}
-                        onInputChange={this.debouncer}
+                        onInputChange={this.handleInputChange}
                         labelKey={'name'}
                         paginate={false}
                         emptyLabel={''}
                         minLength={2}
                         renderMenu={(results, menuProps) => {
                             return(
                                 <Menu {...menuProps}>
-                                    {results.map((result, index) => (
-                                        <TypeaheadMenuItem result={result} index={index} key={index}/>
-                                    ))
-                                    }
-                                </Menu>        
-                            )
+                                    {(this.state.fetched)
+                                    ?
+                                        (results.length)
+                                        ?
+                                            results.map((result, index) => (
+                                                <TypeaheadMenuItem result={result} index={index} key={index}/>
+                                            ))
+                                        :
+                                            <div style={{padding: '16px', userSelect: 'none', cursor: 'default'}}>No results</div>
+                                    : null}
+                                </Menu>
+                            );
                         }}
                     />
+                    {(this.state.fetching)
+                    ?
+                        <CircularProgress
```

Comment:
Can you set the color of the progress indicator to #4598bf? Right now it looks like its using the MUI default color

Reply:
Sounds good!

Reply:
This should be good to go now.